movieapp/src/components/SortElement/SortElement.tsx

```diff
@@ -26,11 +28,11 @@ export function SortElement(): JSX.Element {
   return (
     <select>
       {movieSortCriterias.map((element) => (
-        <>
-          <option key={element.id} value={element.link}>
+        <Fragment key={element.id}>
```

Comment:
what is it for? 

fragments exists to handle situations when u neet to wrap two siblings in one react component e.g.: 
```
<>
  <div>Hi</div>
  <div>Man</div>
</>
```
but not one child

Reply:
fixed